src/client/Microsoft.Identity.Client/Platforms/Android/EmbeddedWebview/AuthenticationAgentActivity.cs

```diff
@@ -26,25 +25,27 @@ protected override void OnCreate(Bundle bundle)
             base.OnCreate(bundle);
             // Create your application here
 
-            WebView webView = new WebView(ApplicationContext);
-            var relativeLayout = new RelativeLayout(ApplicationContext);
-            webView.LayoutParameters = new RelativeLayout.LayoutParams(RelativeLayout.LayoutParams.MatchParent, RelativeLayout.LayoutParams.MatchParent);
+            WebView webView = new WebView(this);
+            var relativeLayout = new RelativeLayout(this);
+            webView.LayoutParameters = new RelativeLayout.LayoutParams(global::Android.Views.ViewGroup.LayoutParams.MatchParent, global::Android.Views.ViewGroup.LayoutParams.MatchParent);
```

Comment:
why do you need to quality it fully?

Reply:
this was a VS suggestion

---
In reply to: [353459593](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1531#discussion_r353459593) [](ancestors = 353459593)

Reply:
I agree with @jmprieur - this makes the code hard to read.

Reply:
ok..i can revert it back

---
In reply to: [353950834](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1531#discussion_r353950834) [](ancestors = 353950834)